scipy/stats/_distribution_infrastructure.py

```diff
@@ -3795,22 +3849,27 @@ def _get_domain_info(info):
 
 
 def _make_distribution_custom(dist):
-    parameters = []
-
-    for name, info in dist.parameters.items():
-        domain_info, typical = _get_domain_info(info)
-        domain = _RealDomain(**domain_info)
-        param = _RealParameter(name, domain=domain, typical=typical)
-        parameters.append(param)
+    dist_parameters = (
+        dist.parameters if isinstance(dist.parameters, tuple) else (dist.parameters, )
+    )
+    parameterizations = []
+    for parameterization in dist_parameters:
```

Comment:
Interesting... I was going to comment about a confusing variable name, but I see why it's named this way. The attribute `parameters` of the custom distribution class really takes the place of `parameterization` in the infrastructure itself. 

Do you think the attribute should still be named `parameters` or should it be `parameterization`? Probably fine to keep it `parameters`, but address it as a comment in the code?

Reply:
Yeah, I struggled a little bit naming things here. I agree with keeping it as `parameters` and addressing it with a code comment. `parameters`  seems reasonable from a user-perspective.